README.md

```diff
@@ -1,2 +1,11 @@
 # mappers-go
 KubeEdge Device Mappers written in go, it works for KubeEdge 1.4 version and later.
+
+## Construction method
+### 1 mappers
```

Comment:
What's meaning of `Construction method` here? Seems there's no content under mappers title.

It's better not to use the number like `1` in markdown